fusor-ember-cli/app/mixins/validated-input-mixin.js

```diff
@@ -53,9 +53,11 @@ export default Ember.Mixin.create({
     // this action is triggered on key-up. however, for password fields only, we don't want
     // to show validation errors after every key stroke unless 8 characters have been entered
     showValidationErrorsKeyUp() {
-      if (!(this.get('isPassword') && this.get('value.length') < 8)) {
-        this.set("showValidationError", true);
-      }
+      let self = this;
+      // delay showing validation for 1 second
+      Ember.run.later((function() {
+        self.set("showValidationError", true);
+      }), 2000);
```

Comment:
We should check with UX.  I was imagining us not displaying any validations while they were typing and waiting until they were done before telling them they did something wrong.  For that to work we'd have to save the timer and cancel it on key-down.  Something like:

```javascript
showValidationErrorsKeyDown() {
  let showValidationTimer = this.get('showValidationTimer');

  if (showValidationTimer) {
    Ember.run.cancel(showValidationTimer);
  }

  this.set("showValidationError", false);
  showValidationTimer = Ember.run.later(() => this.set("showValidationError", true), 1000);
  this.set('showValidationTimer', showValidationTimer);
},
```

We should ask UX if they prefer delaying from start of typing and displaying while they type or not displaying while they type and showing after they stop.  If it's after they stop, we'd cut down the delay.